draft-irtf-cfrg-vdaf.md

```diff
@@ -682,25 +682,19 @@ before they can begin preparing inputs for aggregation.
     | prep_init |  | prep_init |       | prep_init |
     +-----------+  +------------+      +-----------+
       |              |              ...  |             \
-      V              V                   V             |
-    +-----------+  +-----------+       +-----------+   |
-    | prep_next |  | prep_next |       | prep_next |   |
-    +-----------+  +-----------+       +-----------+   |
-      |              |              ...  |             |
       V              V                   V             | x ROUNDS
     +----------------------------------------------+   |
     | prep_shares_to_prep                          |   |
     +----------------------------------------------+   |
                      |                                 |
       +--------------+-------------------+             |
       |              |              ...  |             |
-      V              V                   V             /
+      V              V                   V             |
+    +-----------+  +-----------+       +-----------+   |
+    | prep_next |  | prep_next |       | prep_next |   |
+    +-----------+  +-----------+       +-----------+   |
+      |              |              ...  |             /
```

Comment:
Nit: I would adjust this so that the `...` is vertically centered between the boxes

Reply:
Done.